core/lpms.go

```diff
@@ -131,6 +133,19 @@ func (l *LPMS) HandleHLSPlay(
 	l.vidPlayer.HandleHLSPlay(getMasterPlaylist, getMediaPlaylist, getSegment)
 }
 
+func (l *LPMS) RTMPToHLS(s *segmenter.FFMpegVideoSegmenter, ctx context.Context, cleanup bool) error{
+	var err error
+	for i:=0; i < RETRY_COUNT; i++ {
+		err = s.RTMPToHLS(ctx, true)
+		if err == nil {
+			break
+		} else if i < RETRY_COUNT {
+			glog.Errorf("Error Invoking Segmenter: %v, Retrying", err)
```

Comment:
We normally use a [backoff module](https://github.com/cenkalti/backoff) but maybe here it's OK to put a small sleep before retrying, say 500ms.

Reply:
Added 500ms Sleep